internal/usbgadget/usbgadget.go

```diff
@@ -172,6 +167,12 @@ func (u *UsbGadget) Close() error {
 	u.kbdAutoReleaseTimers = make(map[byte]*time.Timer)
 	u.kbdAutoReleaseLock.Unlock()
 
+	// Cancel keyboard state context
```

Comment:
TODO This looks like a merge error... no reason that this block should be done twice